.github/workflows/template_gitops.yml

```diff
@@ -9,6 +9,10 @@ on:
       docker-build-target:
         required: false
         type: string
+      docker-build-provenance:
+        required: false
+        type: string
+        default: 'false'
```

Comment:
https://github.com/Staffbase/gitops-github-action/pull/66/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R38

should it be false?

Reply:
no, we want `false` as our default because artifactory doesn't support provenance information yet. this causes the error we just discussed

Reply:
some of our teams not using the github action templates. I guess they should be informed that they need to set that to false or we should also change that in the gitops repo

Reply:
> some of our teams not using the github action templates. I guess they should be informed that they need to set that to false or we should also change that in the gitops repo

https://staffbasehq.slack.com/archives/C03UKRYDV50/p1686054985816079